src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -367,6 +367,17 @@ public function get_items( $request ) {
 			add_filter( 'post_password_required', array( $this, 'check_password_required' ), 10, 2 );
 		}
 
+		$parent_ids = array();
+		foreach ( $query_result as $post ) {
+			if ( $post->post_parent > 0 ) {
```

Comment:
This can be simplified as:

```suggestion
			if ( $post->post_parent ) {
```

Due `0` would evaluate to `false`

Reply:
This also this also ensures that the value is a positive int. So is better as it is.